ingq/illust/application/illust_service.py

```
@@ -36,3 +48,16 @@ def get_illust_item_by_story_id(
             return None
 
         return IllustMapper.illustvo_to_illust_item(illust)
+
+    def get_illust_item_list_by_book(
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)